packages/core/src/domain/configuration/endpointBuilder.ts

```diff
@@ -31,6 +31,10 @@ export function createEndpointBuilder(
       const parameters = buildEndpointParameters(initConfiguration, trackType, configurationTags, api, payload)
       return buildUrlWithParameters(parameters)
     },
+    rawParameters: {
+      initConfiguration,
+      configurationTags,
+    },
```

Comment:
This is controversial and will probably won't make it. 
@aymericDD, happy to discuss other ways to get access to these parameters at the "send" level.
See `exportToJSONIntake` function to see how its used.

Reply:
Discussed:  remove initConfig, set `tags` directly.